### PR TITLE
feat(hatcheries): ignoreJobWithNoRegion conf

### PR DESCRIPTION
**Pull request by yesnault**
Ignore job without a region prerequisite if ignoreJobWithNoRegion=true

Signed-off-by: Yvonnick Esnault <yvonnick.esnault@corp.ovh.com>

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#13481.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔ 
* Package
  * Docker ✔ 
  * Docs ✔ 
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✘  
  * Standalone tests 1 -   
  * Standalone tests 2 -   
  * Standalone tests 3 -

File: sdk/hatchery/hatchery.go [hunk 1]
```diff
@@ -220,7 +220,19 @@ func Create(ctx context.Context, h Interface) error {
 			// Check at least one worker model can match
 			var chosenModel *sdk.Model
 			var canTakeJob bool
-			if isWithModels {
+
+			var containsRegionRequirement bool
+			for _, r := range workerRequest.requirements {
+				switch r.Type {
+				case sdk.RegionRequirement:
+					containsRegionRequirement = true
```
Comment (richardlt) [inline sdk/hatchery/hatchery.go:228]: add a break ;)

Comment (yesnault) [reply sdk/hatchery/hatchery.go:228]: @bnjjj I see you :-D

Comment (bnjjj) [reply sdk/hatchery/hatchery.go:228]: Thanks @richardlt for taking this big responsibility 😅

**Pull request merged**